languages/python/sqlalchemy-oso/tests/test_roles.py

```diff
@@ -325,7 +325,7 @@ def test_get_resource_users_by_role(test_db_session, abbey_road, vocalists):
     # Test RepoRoles
     users = (
         test_db_session.query(User)
-        .join(RepositoryRole)
+        .join(RepositoryRole, RepositoryRole.user_id == User.user_id)
```

Comment:
Why is this join condition needed? SQLAlchemy should auto generate these.

Reply:
https://github.com/osohq/oso/pull/879#discussion_r638235911